src/sn_utils.c

```diff
@@ -974,6 +974,7 @@ static int process_udp(n2n_sn_t * sss,
 	if(comm) {
 	  strncpy(comm->community, (char*)cmn.community, N2N_COMMUNITY_SIZE-1);
 	  comm->community[N2N_COMMUNITY_SIZE-1] = '\0';
+	  comm->is_federation = IS_NO_FEDERATION;
```

Comment:
As this code is similar to sn.c perhaps is better to create an initialization function?

Reply:
I've created `comm_init()` function that intialises some fields of a new community